airflow/contrib/kubernetes/worker_configuration.py

```diff
@@ -224,6 +224,14 @@ def make_pod(self, namespace, worker_uuid, pod_id, dag_id, task_id, execution_da
         annotations = {
             'iam.cloud.google.com/service-account': gcp_sa_key
         } if gcp_sa_key else {}
+        labels = {
+                'airflow-worker': worker_uuid,
+                'dag_id': dag_id,
+                'task_id': task_id,
+                'execution_date': execution_date,
```

Comment:
This could become a cardinality issue for Prometheus metrics as it would be unique every run.